test/test_sparse.py

```diff
@@ -402,7 +411,11 @@ def test_ctor_size_checks(self, device, dtype):
 
     @dtypes(*floating_and_complex_types_and(torch.float16, torch.bfloat16))
     @unittest.skipIf(TEST_WITH_CROSSREF, "generator unsupport triggers assertion error")
-    def test_to_dense(self, device, dtype):
+    @gradcheck_semantics()
+    @parametrize("fast_mode", [subtest(False, name='slow'), subtest(True, name='fast')])
+    def test_to_dense_with_gradcheck(self, device, dtype, gradcheck, fast_mode):
+        if not gradcheck.masked and fast_mode == 'slow':
+            self.skipTest(f'FIXME: to_dense_backward supports masked semantics only')
```

Comment:
This is the fixing changeset relative to https://github.com/pytorch/pytorch/pull/94714

Reply:
closing as out-of-date